src/main/java/io/luna/game/model/def/EquipmentDefinition.java

```diff
@@ -161,8 +161,8 @@ public EquipmentDefinition(int id, int index, boolean twoHanded, boolean fullBod
         this.twoHanded = twoHanded;
         this.fullBody = fullBody;
         this.fullHelmet = fullHelmet;
-        this.requirements = ImmutableList.copyOf(requirements);
-        this.bonuses = ImmutableList.copyOf(Ints.asList(bonuses));
+        this.requirements = List.of(requirements);
+        this.bonuses = bonuses.clone();
```

Comment:
Would rather not use clone() as its a badly designed portion of the Java API, `Arrays.copyOf` would be better

Reply:
`clone()` for arrays of primitives is actually well-defined; however, I read into it just now, and `Arrays#copyOf` looks like it's faster than `clone()` for small arrays, as `clone()` returns an `Object` which requires a cast.

Regardless, I planned to remove the calls to `clone()` entirely if the getters themselves can be replaced with methods that perform specific operations on the array.